lib/project_types/script/messages/messages.rb

```diff
@@ -117,6 +119,9 @@ module Messages
             Usage: {{command:%s enable --API_key=<API_key> --shop_domain=<my_store.myshopify.com>}}
           HELP
 
+          info: "A script remains enabled until you manually disable it. To disable the script, "\
+                "either run the 'disable' command or turn the script off in the store admin. ",
+
```

Comment:
Also, we can't assume the Script can be turned off in the Store Admin, as it depends on whether the App implements that feature, which might not be the case.  The alternative to disabling it, will be the unimplemented `delete` [feature](https://github.com/Shopify/script-service/issues/849).

Reply:
Ok, that's fair @ErinRenNumber1. I was keying off what info Kate originally provided. We could just reduce it to say "A script always remains enabled until you manually disable the 'disable' command." Would that be enough of a tweak to resolve that concern?